docker-compose.yml

```diff
@@ -1,8 +1,15 @@
 ---
 version: '3.7'
 services:
+  ch:
```

Comment:
use external ClickHouse (force AMD64), as DBaaS fails with one that is running in container.